src/adapter/src/coord.rs

```diff
@@ -2994,6 +2995,7 @@ pub async fn load_remote_system_parameters(
 ) -> Result<Option<BTreeMap<String, OwnedVarInput>>, AdapterError> {
     if let Some(system_parameter_sync_config) = system_parameter_sync_config {
         tracing::info!("parameter sync on boot: start sync");
+        const FRONTEND_SYNC_TIMEOUT: Duration = Duration::from_secs(30);
```

Comment:
I'm curious about people's thoughts on whether this should be a LaunchDarkly flag itself. Moving this into LD creates a bit of an interesting bootstrap issue where this controls how we sync with LD but is itself stored in LD.

Personally I think it's better off as a constant left out of LD. If the value is set too low, then increasing it in LD will have no effect because it will never be able to sync and get the new value. If the value is set too high, then lowering it won't take affect until the next restart, which is usually the next release.

Reply:
Hardcoding it seems fine to me. Another reasonable option would be a cli flag, which at least gives us an escape hatch without cutting a new image (here, I'm more picturing editing some envd's definition in k9s over getting something through cloud CI)

Reply:
I agree, putting it in LD seems more confusing/weird than it's worth.

Reply:
The CLI flag sounds perfect actually, I'll update it to use that.

Reply:
Updated.